common/src/profile.rs

```diff
@@ -20,108 +22,22 @@ use crate::keys;
 /// Environment var that sets the radicle home directory.
 pub const RAD_HOME: &str = "RAD_HOME";
 
-/// Create a new profile.
-pub fn create<C: Crypto>(home: impl Into<LnkHome>, crypto: C) -> Result<(Profile, PeerId)>
-where
-    C::Error: fmt::Debug + fmt::Display + Send + Sync + 'static,
-    C::SecretBox: Serialize + DeserializeOwned,
-{
-    let home = home.into();
-    let profile = Profile::new(&home)?;
-
-    Profile::set(&home, profile.id().clone())?;
-
-    let key = SecretKey::new();
-    let mut store: FileStorage<C, PublicKey, SecretKey, _> =
-        FileStorage::new(&profile.paths().keys_dir().join(keys::KEY_FILE), crypto);
-
-    store.put_key(key.clone())?;
-    Storage::open(profile.paths(), key.clone())?;
-
-    Ok((profile, PeerId::from(key)))
-}
-
-/// Get the radicle home.
-pub fn home() -> LnkHome {
-    let home = env::var(RAD_HOME)
-        .map(|h| LnkHome::Root(path::Path::new(&h).to_path_buf()))
-        .ok();
-
-    home.unwrap_or_default()
-}
-
 /// Get the default profile. Fails if there is no profile.
 pub fn default() -> Result<Profile, Error> {
     let error = args::Error::WithHint {
         err: anyhow!("Could not load radicle profile"),
         hint: "To setup your radicle profile, run `rad auth`.",
     };
 
+    // TODO(dave): what to do with this?
     let not_active_error = args::Error::WithHint {
         err: anyhow!("Could not load active radicle profile"),
         hint: "To setup your radicle profile, run `rad auth --init`.",
     };
 
-    match Profile::active(&home()) {
-        Ok(Some(profile)) => Ok(profile),
-        Ok(None) => Err(not_active_error.into()),
+    match Profile::load() {
+        Ok(profile) => Ok(profile),
+        //Ok(None) => Err(not_active_error.into()),
         Err(_) => Err(error.into()),
```

Comment:
Thoughts re this?

Reply:
There's a NotFound error that you can match on to return a different message

Reply:
`args::Error` has these 3 variants: `Help`, `Usage`, and `WithHint` ([source](https://github.com/radicle-dev/radicle-cli/blob/master/common/src/args.rs))